MetaMask/metamask-extension
# refactor: removed the whats new modal for portfolio

## **Description**

Removed the "What's new" notification modal for the Portfolio. This is being removed because it was requested.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24566?quickstart=1)

## **Related issues**

Fixes: [1150](https://consensyssoftware.atlassian.net/jira/software/projects/MMPD/boards/468?selectedIssue=MMPD-1150)

## **Manual testing steps**

1. Download MM extension
2. On a fresh install, scroll down to the "what's new" notification modal
3. Check that the Portfolio notifications modal is no longer rendering

## **Screenshots/Recordings**

### **Before**
<img width="369" alt="Screenshot 2024-05-16 at 9 06 39 AM" src="https://github.com/MetaMask/metamask-extension/assets/142838415/b1f395d9-d9a7-409c-938a-2c4c773a0748">

### **After**

https://www.loom.com/share/451bc9aee9024097928a4ca7464e80e7

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
